src/cartridges/int_adyen_overlay/cartridge/scripts/config/blockedPaymentMethods.json

```diff
@@ -1,5 +1,6 @@
 {
   "blockedPaymentMethods": [
+    "ideal",
```

Comment:
This still needs to be removed before I can approve

Reply:
Removed!